project/driver_controls/inc/input_event.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+// Define common typedefs for the various FSMs to use.
+
+#include "event_queue.h"
+
+// Typedef for the FSM arbitration functions
+typedef bool (*InputEventCheck)(const Event *e);
+
+// ID definitions for the driver input events.
+typedef enum {
+  INPUT_EVENT_NONE = 0,                     // Pedal         00
```

Comment:
Those comments were initially created as a quick reference for debugging the FSMs, but they aren't really that necessary anymore.
Also, prefixing the event names does seem like a good idea for organizing the events.